checks.d/spark.py

```diff
@@ -263,6 +267,11 @@ def _get_running_apps(self, instance, tags):
             cluster_mode = SPARK_YARN_MODE
 
         if cluster_mode == SPARK_STANDALONE_MODE:
+            # check for PRE-20
+            pre20 = instance.get(SPARK_PRE_20_MODE)
```

Comment:
I'd use `_is_affirmative()` here (you'll have to import it)...
```
pre20 = _is_affirmative(instance.get(SPARK_PRE_MODE, False))
```
By doing this we achieve two things, if the key doesn't exist (ie. it hasn't been set in the YAML) we'll default to false, and we can also handle stuff like "yes", "no", etc if it is set. You won't need to do the validation like you do in the next couple of lines.